client/clusters/clusters_client.go

```diff
@@ -62,7 +62,7 @@ func (a *Client) DeleteSingleCluster(params *DeleteSingleClusterParams) (*Delete
 		PathPattern:        "/clusters/{name}",
 		ProducesMediaTypes: []string{"application/json"},
 		ConsumesMediaTypes: []string{"application/json"},
-		Schemes:            []string{"http", "https"},
+		Schemes:            []string{"https"},
```

Comment:
after looking at the other patch (enabling tls), i'm not sure why this generated code is now attempting to remove the http option. i wonder if we are having a swagger-codegen version mismatch or something.